lang/java/reef-common/src/main/java/org/apache/reef/driver/ContextAndTaskSubmittable.java

```diff
@@ -60,5 +60,4 @@
   void submitContextAndServiceAndTask(final Configuration contextConfiguration,
                                       final Configuration serviceConfiguration,
                                       final Configuration taskConfiguration);
-
```

Comment:
It does not worth to make another push for the apace line. Even it is format change only, it is a valid change, right? In C# code, style check requires to remove those extra lines. Java may not require it. If you really care, please remove it when you merge. Thanks!